### PR TITLE
Add new Vision Transformer tutorial using experimental workflow interface

### PR DESCRIPTION
**Pull request by kminhta**
This tutorials demonstrates how to set up a basic OpenFL workload using the experimental workflow interface to fine-tune a vision transformer model on the medmnist dataset.

This notebook makes use of standard pytorch practices and leverages HuggingFace's transformer library to build a vision transformer with a pretrained backbone and custom classification head for an image classification workload using medmnist.

It is heavily modeled after the 101_MNIST workflow tutorial

### CODE REVIEW

File: openfl-tutorials/experimental/Vision_Transformer/Workflow_Interface_101_Vision_Transformer.ipynb
Comment (psfoley) [inline openfl-tutorials/experimental/Vision_Transformer/Workflow_Interface_101_Vision_Transformer.ipynb:1]: Could you renumber this example to something like `Workflow_Interface_102_Vision_Transformer.ipynb`?

**Comment by psfoley** [comment]
Hi @kta-intel! This PR looks good. Once the number of the example is changed I think this is ready to merge.

File: openfl-tutorials/experimental/Vision_Transformer/Workflow_Interface_101_Vision_Transformer.ipynb
Comment (kminhta) [reply openfl-tutorials/experimental/Vision_Transformer/Workflow_Interface_101_Vision_Transformer.ipynb:1]: sure, all set!

File: openfl-tutorials/experimental/Vision_Transformer/Workflow_Interface_102_Vision_Transformer.ipynb [hunk 1]
```diff
@@ -0,0 +1,690 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "79a3d106",
+   "metadata": {},
+   "source": [
+    "# Workflow Interface 101: \n",
```
Comment (psfoley) [inline openfl-tutorials/experimental/Vision_Transformer/Workflow_Interface_102_Vision_Transformer.ipynb]: Update reference to 102

File: openfl-tutorials/experimental/Vision_Transformer/Workflow_Interface_102_Vision_Transformer.ipynb [hunk 2]
```diff
@@ -0,0 +1,690 @@
...
+    "# Vision Transformer for Image Classification using MedMNIST\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/intel/openfl/blob/develop/openfl-tutorials/experimental/vision-transformer/workflow-interface-vision-transformer.ipynb)"
```
Comment (psfoley) [inline openfl-tutorials/experimental/Vision_Transformer/Workflow_Interface_102_Vision_Transformer.ipynb]: Colab link should be updated

**Comment by manuelhsantana** [comment]
When I install `!pip install -r requirements_vision_transformer.txt` in a clean environment with only openFL, I got an error with torch version:
ERROR: Could not find a version that satisfies the requirement torch==2.0.1+cpu (from versions: 1.4.0, 1.5.0, 1.5.1, 1.6.0, 1.7.0, 1.7.1, 1.8.0, 1.8.1, 1.9.0, 1.9.1, 1.10.0, 1.10.1, 1.10.2, 1.11.0, 1.12.0, 1.12.1, 1.13.0, 1.13.1, 2.0.0, 2.0.1, 2.1.0, 2.1.1, 2.1.2)
ERROR: No matching distribution found for torch==2.0.1+cpu

**Review by manuelhsantana — APPROVED** [review]
LGTM!
I ran the workflow from the beginning to the end and it ran without any problems!
Great Work Kevin!

**Pull request merged**